cypress/e2e/ui/Settings/Application-Settings/schedule.cy.js

```diff
@@ -0,0 +1,348 @@
+/* eslint-disable no-undef */
+
+function selectConfigMenu(configuration = 'Add a new Schedule') {
+  cy.get('#miq_schedule_vmdb_choice').click();
+  cy.get(`ul[aria-label="Configuration"] [title="${configuration}"]`).click();
+}
+
+function addSchedule() {
+  selectConfigMenu();
+  // Checks if Save button is disabled initially
+  cy.contains(
+    '#main-content .bx--btn-set button[type="submit"]',
+    'Save'
+  ).should('be.disabled');
+  // Adding data
+  cy.get('input#name').type('Test name');
+  cy.get('input#description').type('Test description');
+  cy.get('input[type="checkbox"]#enabled').check({ force: true });
+  cy.get('select#action_typ').select('VM Analysis');
+  cy.get('select#filter_typ').select('A single VM');
+  cy.get('select#timer_typ').select('Hourly');
+  cy.get('select#timer_value').select('1 Hour');
+  cy.get('input[role="combobox"]#time_zone').click();
+  cy.contains('[role="option"]', '(GMT-10:00) Hawaii')
+    .should('be.visible')
+    .click();
+  cy.get('input#start_date').type('06/30/2025', { force: true });
+  cy.get('input#start_time').type('11:23');
+  // Checks if Save button is enabled once all required fields are filled
+  cy.contains('#main-content .bx--btn-set button[type="submit"]', 'Save')
+    .should('be.enabled')
+    .click();
+}
+
+function deleteSchedule(scheduleName = 'Test name') {
+  // Selecting the schedule
+  cy.contains('li.list-group-item', scheduleName).click();
+  cy.on('window:confirm', (text) => {
+    expect(text).to.eq(
+      'Warning: This Schedule and ALL of its components will be permanently removed!'
+    );
+    return true;
+  });
+  selectConfigMenu('Delete this Schedule from the Database');
+  cy.get('#main_div #flash_msg_div .alert-success').contains(
+    `Schedule "${scheduleName}": Delete successful`
+  );
+}
+
+function invokeCleanupDeletion() {
+  // Iterate and clean up any leftover schedules created during the test
+  cy.get('li.list-group-item').each(($el) => {
+    const text = $el?.text()?.trim();
+    if (text === 'Test name') {
+      deleteSchedule();
+      return false;
+    }
+    if (text === 'Dummy name') {
+      deleteSchedule('Dummy name');
+      return false;
+    }
+    return true;
+  });
+}
+
+function goToAppSettings() {
+  cy.intercept('GET', '/api/notifications').as('getNotifications');
+  cy.intercept(
+    'GET',
+    '/api/notifications?expand=resources&attributes=details&sort_by=id&sort_order=desc&limit=100'
+  ).as('getExpandedNotifications');
+  cy.menu('Settings', 'Application Settings');
+  cy.wait('@getNotifications');
+  cy.wait('@getExpandedNotifications');
+}
+
+describe('Automate Schedule form operations: Settings > Application Settings > Settings > Schedules > Configuration > Add a new schedule', () => {
+  beforeEach(() => {
+    cy.login();
+    goToAppSettings();
+    cy.intercept('POST', '/ops/tree_select?id=xx-msc&text=Schedules').as('getSchedules');
+    cy.get('[title="Schedules"]').click();
+    cy.wait('@getSchedules');
```

Comment:
I think there's are the right way to improve test reliability by waiting on the various requests to complete before moving on.  We might need to refactor common `intercepts` as helper methods.  Keep this in mind as you find that you need to do this over and over again.

We have similar concerns where we don't use unique class, ids, etc. in the DOM.  We should probably change code where possible to make it much easier to find elements in the DOM for testing.